src/app/auth/auth.component.ts

```diff
@@ -124,6 +118,8 @@ export class AuthComponent implements OnInit, OnDestroy {
           }
           this.authService.storeOAuthAccessToken(data.token);
           this.logger.info('Sucessfully obtained access token');
+          this.router.navigate(['/']);
```

Comment:
Is this needed?

Reply:
Oh just tried deleting the line.
Turns out it's not needed after all.
Originally I just put that line during debugging to ensure it goes back to the homepage.
I'll remove this, thanks for the input!